packages/inula-adapter/tests/vue/dynamicComponent.test.tsx

```
@@ -0,0 +1,68 @@
+          Hello, DOM elements!
+        </DynamicComponent>
+      </DynamicComponent>
+    );
+
+    const divElement = screen.getByTestId('dynamic-div');
+    expect(divElement.tagName).toBe('DIV');
+
+    const spanElement = screen.getByTestId('dynamic-span');
+    expect(spanElement.tagName).toBe('SPAN');
+
+    // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)